resources/assets/js/fotoalbum/FotoAlbum.ts

```diff
@@ -1,13 +1,27 @@
 import $ from 'jquery';
-import EventEmitter from 'events';
+import {EventEmitter} from 'events';
 import {basename, dirname, redirect, reload, selectText} from '../util';
 
 import 'jgallery/dist/js/jgallery';
 
-class FotoAlbum extends EventEmitter {
-	container = null;
+declare global {
+	interface JQuery {
+		jGallery(options: any): void;
```

Comment:
Van jGallery zijn geen types te vinden op npm, dus options is een `any` hier, had geen zin om de types te definieren.